### ISSUE TITLE
Automate setting a default keyboard shortcut in OSX

### ISSUE THREAD

**Issue by cknadler** [cknadler/vim-anywhere#1]
Upon running the install script, a default keyboard shortcut should be set for invoking vim-anywhere on OSX.

I have absolutely no idea how to do this.

**Comment by cknadler** [comment]
Would be over the moon if anyone could figure this one out. I never could quite get it to work on OSX.

**Comment by jgrasser** [comment]
Enabling the shortcut via System Preferences results in this entry appearing in the Mac OS X user defaults system : 
``` 
$ defaults read pbs
{
    NSServicesStatus =     {
        "org.vim.MacVim - MacVim/New MacVim Buffer With Selection - openSelection" =         {
            "key_equivalent" = "@^v";
        };
    };
}
```

You can add this manually by running the following:
```
$ defaults write pbs NSServicesStatus '{                                                                      
        "org.vim.MacVim - MacVim/New MacVim Buffer With Selection - openSelection" =         {
            "key_equivalent" = "@^v";
        };
  }'
```

Is this an acceptable solution? I am not entirely familiar with MacOSX so this answer might be incomplete, or incorrect.

**Comment by cknadler** [comment]
Following up in #74.

**Comment by cknadler** [comment]
Closed by #74
